packages/opentelemetry-host-metrics/src/BaseMetrics.ts

```diff
@@ -17,7 +17,7 @@
 import * as api from '@opentelemetry/api';
 import * as metrics from '@opentelemetry/sdk-metrics';
 
-import { VERSION } from './version';
+import { PACKAGE_VERSION } from './version';
```

Comment:
There is also a `const DEFAULT_NAME = '@opentelemetry/host-metrics';` that can be updated to `PACKAGE_NAME`

Reply:
good catch 👍  fixed